src/renderer/assets/styles/components/blocks.css.d.ts

```diff
@@ -5,9 +5,9 @@ declare const styles: {
   readonly "block_full": string;
   readonly "block_full_wrapper": string;
   readonly "block_full_close": string;
+  readonly "block_full_update": string;
   readonly "block_dashed": string;
   readonly "___DEBUG___COMPONENTS_BLOCKS_CSS": string;
-  readonly "___DEBUG___PARTIALS_VARIABLES_CSS": string;
```

Comment:
Interesting to see that the auto-generated "typed CSS" now excludes `__DEBUG__xxx` classes  which I assume where transitively included by the build plugin (PostCSS). Never mind, as long as linting and compiling works fine :) (the resulting compiled / bundled CSS stylesheets should still contain the DEBUG classes, they are useful to troubleshoot CSS module duplicates and cascade ordering issues)